### ISSUE TITLE
Project status & maintanance

### ISSUE THREAD

**Issue by hinell** [rohit-px2/nvui#157]
It seems like this project isn't supported anymore. Any news on that?

**Comment by rohit-px2** [comment]
Yeah, I've been pretty busy with work, school, and other things so I haven't had the time/motivation to work on this project for a while. Not sure if I'll come back to this since I'm working on other things at the moment.
